nix-tools-regenerate.nix

```diff
@@ -32,7 +32,7 @@ in
     }
     trap cleanup EXIT
 
-    stack-to-nix -o nix stack.yaml > "$dest.new"
+    stack-to-nix --output nix > "$dest.new"
```

Comment:
```suggestion
    stack-to-nix --output nix
```

Reply:
I don't think `stack-to-nix` generates anything on STDOUT anymore.

But then the two `mv` and `echo` lines below make no sens anymore either.